CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## Version 1.1.2 - Internal release
+- Added support for Python 3.8, 3.9, 3.10 (experimental), 3.11 (experimental) 
```

Comment:
I was told by Alex Bourret that python 2.7 is now deprecated, we should add that here

Reply:
done